src/components/character-count/template.test.js

```diff
@@ -0,0 +1,273 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-env jest */
+
+import axe from '../../../lib/axe-helper';
+
+import { render, getExamples, htmlWithClassName } from '../../../lib/jest-helpers';
+
+const examples = getExamples('character-count');
+
+const WORD_BOUNDARY = '\\b';
+
+describe('Character count', () => {
+  describe('default example', () => {
+    it('passes accessibility tests', async () => {
+      const $ = render('character-count', examples.default);
+
+      const results = await axe($.html(), {
+        rules: {
+          region: { enabled: false },
```

Comment:
Why is this marked as false?

Reply:
@timsb This is because otherwise accessibility tests fail due to landmarks, i.e. that this component does not included tags such as `<head>` and `<body>`. You can see this also being used in other components such as [here](https://github.com/hmrc/hmrc-frontend/blob/c29953378a5897022ad69ba8a859d2e568f07fce/src/components/user-research-banner/user-research-banner.test.js) and [here](https://github.com/hmrc/hmrc-frontend/blob/15cb948c6c9bb283bf33218088a545b36eec32e6/src/components/account-menu/account-menu.test.js).

Reply:
Ah yes, I thought we'd done this in the past. Makes sense. Turning it off does mean we have to be careful we don't don't break this rule but there shouldn't be many cases where we would.